src/test/java/RenameAPlaylist.java

```diff
@@ -1,53 +1,66 @@
 import org.openqa.selenium.By;
 import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
 import org.openqa.selenium.support.ui.ExpectedConditions;
 import org.testng.Assert;
 import org.testng.annotations.Test;
+import org.openqa.selenium.Keys;
 
 public class RenameAPlaylist extends BaseTest {
 
-    String email = "holostenco.yuliya@gmail.com";
-    String Password = "te$t$tudent";
-    String xpath = "//ul//li[@class='playlist playlist']";
-    String css ="li>input";
-   String  newPlaylistName=" -Best Club music Selection";
+
+    By newPlaylistLocator = By.cssSelector(".playlist:nth-child(3)");
+    By getNewPlaylistTextFieldLocator =By.cssSelector("input[name='name']");
+    String  newPlaylistName= "Best";
 
     @Test
-    public void renameSelectedPlaylist(){
+
+    public void renameSelectedPlaylist() {
         login();
         chooseAPlaylist();
+        nameNewPlaylist();
+        Assert.assertEquals(newPlaylistName, getTextNameNewPlaylist());
 
-wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector(css)));
-        WebElement renamedPlaylist=driver.findElement(By.cssSelector(css));
-        renamedPlaylist.sendKeys(newPlaylistName+"\n");
-        Assert.assertTrue(renamedPlaylist.isDisplayed());
     }
+    public void login() {
+        WebElement emailField = driver.findElement(By.xpath("//input[@placeholder='Email Address']"));
+        waitForElementToBeClickable(emailField).click();
+        emailField.sendKeys(email);
+
+        WebElement passwordField = driver.findElement(By.xpath("//input[@type='password']"));
+        waitForElementToBeClickable(passwordField).click();
+        passwordField.sendKeys(Password);
```

Comment:
For variables, the Java naming convention is to always start with a lowercase letter and then capitalize the first letter of every subsequent word. Variables in Java are not allowed to contain white space, so variables made from compound words are to be written with a lower camel case syntax.

Here are three examples of variables that follow the standard Java naming convention:

```
passwordUser
passwordUserTesrPro
password
```

![image](https://user-images.githubusercontent.com/104761482/200171300-3e8fe118-398e-4289-ad3c-7d7f41f73d07.png)

You can use ⇧F6 on macOS, or Shift + F6 on Windows/Linux, to invoke the renaming refactoring and IntelliJ IDEA will suggest alternative names for your classes, methods and variables. IntelliJ IDEA completes the refactor safely if you select a new name.